runtime/queries/rust/highlights.scm

```diff
@@ -156,16 +156,21 @@
   "macro_rules!"
 
   "let"
-  "ref"
-  "move"
 
-  "dyn"
-  "static"
-  "const"
   "async"
 ] @keyword
 
-(mutable_specifier) @keyword.mut
+(reference_type "&" @keyword.storage)
+(self_parameter "&" @keyword.storage)
+
+[
+  (mutable_specifier)
+  "static"
+  "const"
+  "ref"
+  "move"
+  "dyn"
+] @keyword.storage
```

Comment:
`@keyword.storage.modifier` as per https://www.sublimetext.com/docs/scope_naming.html#storage (we just namespace under keyword)